coderd/users_test.go

```diff
@@ -264,7 +265,12 @@ func TestPostLogout(t *testing.T) {
 	t.Run("Logout", func(t *testing.T) {
 		t.Parallel()
 
-		client := coderdtest.New(t, nil)
+		accessURL, err := url.Parse("http://somedomain.com")
+		require.NoError(t, err)
+
+		client := coderdtest.New(t, &coderdtest.Options{
+			AccessURL: accessURL,
+		})
```

Comment:
Why do we need a custom `AccessURL` to check if multiple cookies are set? We can change `127.0.0.1` to localhost, which is a valid hostname, then we won't need to pass this option.